eo-runtime/src/test/eo/org/eolang/goto-tests.eo

```diff
@@ -44,7 +53,7 @@
               TRUE
       stdout "Finished!"
       i
-    $.equal-to 10
+    $.equal-to 13
```

Comment:
@mximp this is the problem. I suggest to try to use `while` in "imperative" style, as explained in [my recent post](https://news.eolang.org/2022-12-22-declarative-while.html). Or maybe some other hybrid approach. We need to find a solution. Just fixing the test is not a suitable solution.